shared/studio/state/connection.ts

```diff
@@ -320,7 +320,12 @@ export class Connection extends Model({
       let state = this._state;
 
       if (opts.ignoreSessionConfig) {
-        state = setQueryTag(baseOptions.withGlobals(state.globals), "gel/ui");
+        state = setQueryTag(
+          baseOptions
+            .withConfig({apply_access_policies: false})
```

Comment:
does it have a chance to leak this into user-run queries?

Reply:
It shouldn't as we don't use the `userQuery: true` and `ignoreSessionConfig: true` options together anywhere, but I've updated the types to make those options mutually exclusive to be safe.